_events/djangoconus_2023.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: events
+name: "DjangoCon US 2023"
+logo_url: "/img/events/djangocon2023.svg"
+dates: "Mon, Oct 16, 2023 9:00 AM  Fri, Oct 20, 2023 5:00 PM"
+place: "San Diego Marriott of Mission Valley"
+tutorials: "October 8, 2023"
+talks: "October 16-18, 2023"
+sprints: "October 19-20, 2023"
+website: "https://2023.djangocon.us"
+---
+
+We are pleased to announce that the DjangoCon US 2023 conference will take place in Durham, North Carolina from October 11-16, 2023 at the Durham Convention Center.
```

Comment:
11-16 doesn't sound right.

Reply:
Ha, yup. Because it literally says 16-20 above that. Thank you for catching that.